### PR TITLE
SPCTR-3323: Dashboard - 5 star rating in footer area

### PR DESCRIPTION
**Pull request by yash-tingre-10**
### Description
Added a 5-star rating in the footer area to encourage user ratings on WordPress.org based on their experience.

### Screenshots
https://d.pr/i/GLvDar
Task: https://brainstormforce.atlassian.net/browse/SPCTR-3323

### Types of changes
Introducing a new feature, which is the 5-star rating in the footer.
It's an improvement to the existing footer section by adding the rating feature.

### How has this been tested?
I have tested this feature by verifying its functionality in the admin dashboard footer area.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: admin-core/inc/admin-menu.php [hunk 1]
```diff
@@ -436,8 +436,16 @@ public function settings_app_scripts( $localize ) {
 	 *  Add footer link.
 	 */
 	public function add_footer_link() {
-		// translators: HTML entities.
-		return '<span id="footer-thankyou">' . sprintf( __( 'Thank you for using %1$sSpectra.%2$s', 'ultimate-addons-for-gutenberg' ), '<a href="https://wpspectra.com/" class="focus:text-spectra-hover active:text-spectra-hover hover:text-spectra-hover">', '</a>' ) . '</span>';
+		return '<span id="spectra-footer-thankyou" style="font-family: Inter, sans-serif;">' . sprintf(
+			// translators: HTML entities.
+			__(
+				'Enjoyed %1$s? Please leave us a %2$s rating. We really appreciate your support!',
```
Comment (bsf-zanev) [inline admin-core/inc/admin-menu.php:442]: All the text strings should be in the first parameter of sprintf.
```suggestion
			// translators: %1$s: Opening Strong Tag, %2$s: Closing Strong Tag, %3$s Anchor Tag with Star Symbol Codes
			__(
				'Enjoyed %1$sSpectra%2$s? Please leave us a %3$s rating. We really appreciate your support!',
```

File: admin-core/inc/admin-menu.php [hunk 2]
```diff
@@ -436,8 +436,16 @@ public function settings_app_scripts( $localize ) {
...
+				'ultimate-addons-for-gutenberg'
+			),
+			'<strong>Spectra</strong>',
```
Comment (bsf-zanev) [inline admin-core/inc/admin-menu.php:445]:
```suggestion
			'<strong>',
			'</strong>',
```

File: readme.txt [hunk 3]
```diff
@@ -217,6 +217,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 = x.x.x =
 * Improvement: Buttons - Added a new vertical alignment option.
+* Improvement: Added 5 star Spectra footer rating
```
Comment (bsf-zanev) [inline readme.txt:220]: Let's remove the changelog entry for this one, since we don't ned to show the user that this was added IMO.

**Review by bsf-zanev — COMMENTED** [review]
Added a few minor comments.

**Review by AbhijitNage123 — APPROVED** [review]
LGTM

**Review by bsf-zanev — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**